contrib/database/sql/option.go

```diff
@@ -83,3 +86,12 @@ func WithChildSpansOnly() Option {
 		cfg.childSpansOnly = true
 	}
 }
+
+// WithCommentInjection enables injection of tags as sql comments on traced queries.
+// This includes dynamic values like span id, trace id and sampling priority which can make queries
+// unique for some cache implementations. Use WithStaticTagsCommentInjection if this is a concern.
+func WithCommentInjection(mode tracer.SQLCommentInjectionMode) Option {
```

Comment:
```suggestion
func WithSQLCommentInjection(mode tracer.SQLCommentInjectionMode) Option {
```
Even though I like it as short as possible, I think we need to be specific here. Injection is ambiguous otherwise.